### DISCUSSION TITLE
Need collapseOnMiniScreen

### DISCUSSION THREAD

**Discussion by vara-prasad-f9** [rappasoft/laravel-livewire-tables#1351] [General]
I need this collapse on 991px screen

**Comment by lrljoe** [comment]
Are you using Tailwind or Bootstrap as your theme (and which version please).

If you have specific (non-standard) resolutions, then you could tweak your Tailwind config in the meantime (if using Tailwind).

More custom collapsing is in-flight, but isn't presently a high priority item.
